tests/types/alpm-test.json

```diff
@@ -0,0 +1,140 @@
+{
+  "$schema": "https://packageurl.org/schemas/purl-test.schema-1.0.json",
+  "tests": [
+    {
+      "description": "Parse test for <class 'type'> PURL",
+      "test_group": "base",
+      "test_type": "parse",
+      "input": "pkg:alpm/arch/pacman@6.0.1-1?arch=x86_64",
+      "expected_output": {
+        "type": "alpm",
+        "namespace": "arch",
+        "name": "pacman",
+        "version": "6.0.1-1",
+        "qualifiers": {
+          "arch": "x86_64"
+        },
+        "subpath": null
+      },
+      "expected_failure": false,
+      "expected_failure_reason": null
+    },
+    {
+      "description": "Rountrip test for <class 'type'> PURL",
```

Comment:
I think you mean "round trip". This mistake appears several times in this PR.

Reply:
We will fix this and other typos in follow up PRs.

Reply:
https://github.com/package-url/purl-spec/issues/539